packages/ogmios/test/ogmiosToCore/testData.ts

```diff
@@ -628,32 +636,33 @@ export const mockBabbageBlock: Ogmios.Schema.Babbage = {
               addressAttributes: null,
               chainCode: '4d',
               key: 'de46fb9f52d8a229e3cedc60002564a3d23309dbf21dfaabd38485c4538419b2',
-              signature: 'P20IobereBini+jUjqgosOmXYlikyjgVf2KP5N4fMmpO9APEDQNIhk3nclm+mlxgavILKevn9q61lKxAF3coqg=='
+              signature:
+                '3163e1a4cc86af1d07e1930f3059ee20739acb080375d87a80c238bd38db914b3360273ff045d6626fd5c524b5a96f5b36934c5f62678939c49115787b16b9cf'
             },
             {
               addressAttributes: 'mQ==',
               chainCode: '20',
               key: 'fc1a29ed3f0a4c1a4036298831ce117a23b6b714abba83d0a093d0839e193c39',
-              signature: 'PxmE3tWzUZi8/Oqk1fRLJsuOaqWc3nTleLBvk1G9sXVjFiN08ZmYBCPlLd5si35Cv0yhIN2/VXJBuaWbcwgJ/g=='
+              signature:
+                'd94ab1307736a0b3861612da3247bfc20312c6e029609a9359607509bc66bdcecb92adc2da4c5cc42876cb4d35e999140b593af53792d5a39b355f1697c9039c'
             }
           ],
           datums: {},
           redeemers: {
             'spend:0': {
               // eslint-disable-next-line @typescript-eslint/no-loss-of-precision
               executionUnits: { memory: 6_026_711_518_256_058_765, steps: 4_749_668_747_002_318_509 },
-              redeemer:
-                'a34160029f20a3419c204386a48244e21bd936202004a0ffa3a4435aed90040242b07a210523409f414d429b3f0440ffa09f034001ffa222234041b4d8668218da9f2341440302413eff4204e1a1d866821901939f23ffa0'
```

Comment:
These weren't valid redeemer values since the [Hash28ByteBase16()](https://github.com/input-output-hk/cardano-js-sdk/blob/a7f90b7572552e3bb24605ec24482250e0d9e905/packages/core/src/Cardano/util/primitives.ts#L147) requires 56 length, they were with more

Reply:
I think our type is incorrect here. It should be [plutus_data](https://github.com/input-output-hk/cardano-ledger/blob/master/eras/babbage/test-suite/cddl-files/babbage.cddl#L314), we don't have a type for it yet. I suggest for now just make it `export type PlutusData = OpaqueString<'PlutusData'>;` validated same as HexBlob

Reply:
Addressed in 87452b9ae3a7d1f66bab6284d03822a63ed72174